e2etest/bgp_test.go

```diff
@@ -1363,3 +1441,56 @@ func uint32Ptr(n uint32) *uint32 {
 func boolPtr(b bool) *bool {
 	return &b
 }
+
+// Adds the routes that enable communication between both networks that the multi-hop container is connected to for the given Executor.
+func addMultiHopRoutes(exec executor.Executor, execnet string) error {
```

Comment:
I'd move this to a separate package. Container, maybe

Reply:
done